main/reference/zoe-api/zoe-data-types.md

```diff
@@ -63,14 +35,13 @@ You can get information about the contract instance via these methods:
 
 ## Invitation
 
-An **Invitation** is a kind of **[Payment](/reference/ertp-api/payment.md)**. It's a non-fungible eright
-that can be held in **Payments** or **[Purses](/reference/ertp-api/purse.md)**, just like any other
-eright. An **Invitation** **Payment** is a **Payment** holding an **Invitation**.
+An **Invitation** is a **[Payment](/reference/ertp-api/payment.md)** holding a non-fungible eright created by the **[InvitationIssuer](#invitationissuer)**.
+An **Invitation Payment** is a **Payment** holding an **Invitation**.
 
 ## InvitationIssuer
 
-The **InvitationIssuer** is a special type of **[Issuer](/reference/ertp-api/issuer.md)**. The single Zoe
-instance has an **InvitationIssuer** for the entirety of its lifetime. All **Invitations** come from the
+The **InvitationIssuer** is a special type of **[Issuer](/reference/ertp-api/issuer.md)**.
+Zoe has a single **InvitationIssuer** for its entire lifetime. All **Invitations** come from the
 **[Mint](/reference/ertp-api/mint.md)** associated with the Zoe instance's **InvitationIssuer**.
 
 **InvitationIssuer** has all the methods of regular **Issuers**, but the two methods that are most
```

Comment:
> ```md
> two methods that are most
> often used are **[anIssuer.claim()](/reference/ertp-api/issuer.md#anissuer-claim-payment-optamount)**
> and **[anIssuer.getAmountOf()](/reference/ertp-api/issuer.md#anissuer-getamountof-payment)**.
> ```

There is no longer an issuer.claim method.

Reply:
There are still references to it all over this site, so I'm going to keep this in place for whatever work cleans up all of them: #885

Reply:
With that issue filed, resolving here.